src/rproc/InfileMerger.cc

```diff
@@ -70,27 +72,23 @@
 #include "sql/SqlErrorObject.h"
 #include "sql/statement.h"
 #include "util/Bug.h"
+#include "util/Error.h"
 #include "util/IterableFormatter.h"
-#include "util/StringHash.h"
 #include "util/Timer.h"
 
-namespace {  // File-scope helpers
-
-LOG_LOGGER _log = LOG_GET("lsst.qserv.rproc.InfileMerger");
-
 using namespace std;
+namespace util = lsst::qserv::util;
 
-using lsst::qserv::mysql::MySqlConfig;
-using lsst::qserv::rproc::InfileMergerConfig;
-using lsst::qserv::rproc::InfileMergerError;
-using lsst::qserv::util::ErrorCode;
+namespace {
+
+LOG_LOGGER _log = LOG_GET("lsst.qserv.rproc.InfileMerger");
 
 /// @return a timestamp id for use in generating temporary result table names.
 std::string getTimeStampId() {
     struct timeval now;
     int rc = gettimeofday(&now, nullptr);
     if (rc != 0) {
-        throw InfileMergerError(ErrorCode::INTERNAL, "Failed to get timestamp.");
+        throw util::Error(util::ErrorCode::INTERNAL, "Failed to get timestamp.");
```

Comment:
I will look at options. BTW, I eliminated the `typedef` for `InfileMergerError` since I didn't notice any specific attempts to catch this specific exception.